### PR TITLE
FSW-848 Github API Form

### PR DESCRIPTION
**Pull request by garrettdeguchi**
FSW-848 Github API Form

### CODE REVIEW

File: Github_Form_Script/form.ini [hunk 1]
```diff
@@ -0,0 +1,31 @@
+[USER]
+username = your_github_username
+password = your_github_password
```
Comment (vivianng30) [inline Github_Form_Script/form.ini]: Would be nice if the user can enter the password from a prompt or as argument so that password is not stored on file.

**Comment by nguyencnguyen** [comment]
@garrettdeguchi 
Great job!
A couple suggestions for enhancement:
1. add usage/help for the script.
2. give each parameter a name (i.e -u username -p password, etc...)
3. make sure the script continues to support scripting/automation

File: Github_Form_Script/pull_request_form.py [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    user = 'user'
+    merged_at = 'merged_at'
+    merged_commit_sha = 'merge_commit_sha'
+    parents = 'parents'
+    sha = 'sha'
+    base = 'base'
+    ref = 'ref'
+
+class QueryItems(Enum):
+    pull_state_closed = '/pulls?state=closed&per_page=400'
```
Comment (vivianng30) [inline Github_Form_Script/pull_request_form.py]: Not used any more?

File: Github_Form_Script/pull_request_form.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    df_users = 'user'
+
+
+class PRItems(Enum):
+    number = 'number'
+    url = 'url'
+    title = 'title'
+    user = 'user'
+    merged_at = 'merged_at'
+    merged_commit_sha = 'merge_commit_sha'
```
Comment (vivianng30) [inline Github_Form_Script/pull_request_form.py]: Not used any more?  Remove all items no longer used.

File: Automate_Release_Build_Script/icp_automate.py [hunk 4]
```diff
@@ -94,6 +94,8 @@ def convert_files(self, input_file, input_pic_type, input_checksum, input_name_t
         # TODO This portion may be weird as it gives errors on the hex file
         self.keyboard.send_keys("{ENTER}", pause=DEBUG_PAUSE)  # Open
         sleep(SLEEP_TIME)
+        if(self.app.top_window().window_text() == "HEX File Warning"):
```
Comment (nguyencnguyen) [inline Automate_Release_Build_Script/icp_automate.py]: If this change is not related to the GH PR API, I recommend to remove it from this PR.

File: Github_Form_Script/pull_request_form.py [hunk 5]
```diff
@@ -249,20 +244,41 @@ def check_commits(self, input_commits):
...
+                  'This script will output a .csv and .html file based on the repository\n'
+
+    print(help_output)
+
 def main(args):
     logger.info("Pull Request Form Start:")
 
+    password = None
+    username = None
+    for i in range(0, len(args)):
```
Comment (nguyencnguyen) [inline Github_Form_Script/pull_request_form.py]: consider using getopt (import getopt).

File: Github_Form_Script/form.ini (hunk 1 above)
Comment (garrettdeguchi) [reply Github_Form_Script/form.ini]: Changed to -u -p in argument section of script

File: Github_Form_Script/pull_request_form.py (hunk 2 above)
Comment (garrettdeguchi) [reply Github_Form_Script/pull_request_form.py]: Removed unused items

File: Github_Form_Script/pull_request_form.py (hunk 3 above)
Comment (garrettdeguchi) [reply Github_Form_Script/pull_request_form.py]: Removed unused items

File: Automate_Release_Build_Script/icp_automate.py (hunk 4 above)
Comment (garrettdeguchi) [reply Automate_Release_Build_Script/icp_automate.py]: Removed from this PR

File: Github_Form_Script/pull_request_form.py (hunk 5 above)
Comment (garrettdeguchi) [reply Github_Form_Script/pull_request_form.py]: When trying out getopt I did not like how it is necessary to use try except clause if the arguments are not there. I believe this will suffice with similar length in code and readability to getopt
